sklearn/decomposition/tests/test_pca.py

```diff
@@ -532,7 +532,7 @@ def check_pca_float_dtype_preservation(svd_solver):
     assert pca_64.transform(X_64).dtype == np.float64
     assert pca_32.transform(X_32).dtype == np.float32
 
-    assert_allclose(pca_64.components_, pca_32.components_, rtol=1e-4)
+    assert_allclose(pca_64.components_, pca_32.components_, rtol=2e-4)
```

Comment:
maybe a comment as why this tolerance?

supporting platforms which we don't have in our CI doesn't sound logical to me though.

Reply:
Are we okay with conda-forge skipping this test for one of their platforms?

Reply:
I think it's better to increase tolerance rather than ask distributors to skip tests..